### PR TITLE
Detect UITour failure / non-response on what's new page (Fixes #7671)

### LINKED ISSUE #7671: Detect UITour failure / non-response on what's new page
We're trying to evaluate how widespread is https://bugzilla.mozilla.org/show_bug.cgi?id=1557153 where UITour default permission gets lost, so UITour does not respond to events. E.g., Mozilla.UITour.ping(cb) should call back pretty immediately except it never happens with the lost permission.

### PR DESCRIPTION
**Pull request by alexgibson**
## Description
- Attempts to measure the % of people who may have a Firefox profile issue where UITour isn't working as expected.

## Issue / Bugzilla link
#7671

### CODE REVIEW

File: media/js/firefox/whatsnew/whatsnew-69.js [hunk 1]
```diff
@@ -5,6 +5,27 @@
...
+
+        Mozilla.UITour.ping(function() {
+            window.clearTimeout(timeout);
+            trackImpression('UITour Response');
+        });
+
+        function trackImpression(eLabel) {
+            var endTime = performance.now() - startTime;
+
+            window.dataLayer.push({
```
Comment (alexgibson) [inline media/js/firefox/whatsnew/whatsnew-69.js]: @hoosteeno can you advise if this `dataLayer` event looks ok?

Comment (hoosteeno) [reply media/js/firefox/whatsnew/whatsnew-69.js]: Yes, thank you.

**Comment by amychurchwell** [comment]
@hoosteeno does this need to be merged?

**Comment by alexgibson** [comment]
@amychurchwell yes please, this still needs code review.

**Pull request merged**